app/components/publications/publication/translations/translation-requested-panel.js

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { isEmpty } from '@ember/utils';
+import { task } from 'ember-concurrency-decorators';
+
+/**
+ * @argument {requestActivity}
+ */
+export default class PublicationsPublicationTranslationTranslationRequestedPanel extends Component {
+  @tracked isVerifyingDelete = false;
+  @tracked translationActivity;
+
+  constructor() {
+    super(...arguments);
+    this.loadTranslationActivity.perform();
+  }
+
+  get isTranslationActivityFinished() {
+    return !isEmpty(this.translationActivity.endDate);
+  }
+
+  @task
+  *loadTranslationActivity() {
+    this.translationActivity = yield this.args.requestActivity.translationActivity;
+  }
+
+  @action
+  promptDeleteRequest() {
```

Comment:
Waarom "prompt" en niet "verify" of "show" of "open".
Van deze 4 gebruiken we meestal "open", "prompt" slechts 2 keer + deze